tools/make-dist

```diff
@@ -1,18 +1,21 @@
 #!/bin/sh
 
 set -eu
-cd "$(realpath -m "$0"/../..)"
+cd "${0%/*}/.."
```

Comment:
This should be in the first commit, not "stub out xgettext"

Reply:
Thanks